planner/core/logical_plan_builder.go

```diff
@@ -1414,7 +1414,7 @@ func (b *PlanBuilder) resolveWindowFunction(sel *ast.SelectStmt, p LogicalPlan)
 		colMapper:    b.colMapper,
 		outerSchemas: b.outerSchemas,
 	}
-	extractor.curClause = windowClause
```

Comment:
We can remove windowCluase directly?

Reply:
I had changed all the usages of `windowClause` to `fieldList`.

Reply:
@wjhuang2016 windowCluase is not referenced by any places so we can totally remove it.

Reply:
I got it.